nym-vpn-core/crates/nym-gateway-directory/Cargo.toml

```diff
@@ -6,21 +6,22 @@ license.workspace = true
 
 [dependencies]
 async-trait.workspace = true
-chrono = "0.4.38"
+chrono.workspace = true
```

Comment:
this is definitely out of scope (so don't change it now), but why are we pulling in chrono? we're using `time` literally everywhere else : )

Reply:
Ah good to know, I've forgotten the reason why this is used, likely because it's spread here from explorer-api